pkg/api/api.go

```diff
@@ -32,3 +33,7 @@ type PortDriverInfo struct {
 	Protos                  []string `json:"protos"`
 	DisallowLoopbackChildIP bool     `json:"disallowLoopbackChildIP,omitempty"` // since API v1.1.1
 }
+
+type ChildDriverInfo struct {
```

Comment:
This shouldn't be part of the REST API and should be moved to `pkg/network`

Reply:
Thanks for the tip, moved there.